DESCRIPTION.md

```diff
@@ -8,8 +8,11 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v3.3.2(TBD)
+- v4.0.0(Unreleased)
```

Comment:
I think we shouldn't major bump for this change.
```suggestion
- v3.3.2(Unreleased)
```
However; we should pre-announce this and tell users that they should re-create their virtualenvs, manually remove these dependencies.

Reply:
updated to be 3.4.0